### ISSUE TITLE
Docs: how is lasso different from webpack and rollup?

### ISSUE THREAD

**Issue by jwhitfieldseed** [lasso-js/lasso#165]
"[Another Client-side Bundler?](https://github.com/lasso-js/lasso#another-client-side-bundler)" makes it clear how lasso is different from Browserify. 

It doesn't mention other bundlers, however - Rollup and Webpack both have some of the features that differentiate Lasso from Browserify. It would be interesting to know what differentiates Lasso from these projects, and when it makes sense to use Lasso or something else.

Thanks!

**Comment by patrick-steele-idem** [comment]
You are right that we need to do a better job of communicating the differences. I'll try to write up something more detailed later, but at a high level:

- lasso - general purpose asset pipeline with built-in support for JavaScript module bundling
- rollup - just a JavaScript module bundler (it's possible to use rollup with lasso via a lasso plugin)
- webpack - more similar to lasso, but more complex and missing features related to generating HTML markup (among other things). In my opinion, Lasso has much better support for dealing with all types of front-end assets (vanilla JS, fonts, images, etc.) 

Thanks for opening the issue. I do hope to spend some more time on this as soon as I get a chance so I will leave it open.

**Comment by idibidiart** [comment]
Hi,

Thank you for working on Lasso and sharing it with the community. 

I was communicating a list of tools that can perform bundling and code splitting for React apps (and that aren't mentioned in the React docs) and I brought up Lasso as one example. Webpack's author mentioned that he'd be interested to see the detailed comparison. While it's burdensome to make an account of every feature a similar tool has and build a comprehensive analysis, what is people are interested in I think is a high level comparison of the main areas of functionality. 

The community definitely benefits from more competition/alternatives in this area.
